.eslintrc.json

```diff
@@ -13,9 +13,28 @@
   },
   "plugins": ["react", "prettier"],
   "rules": {
-    "prettier/prettier": "error",
-    "quotes": [2, "double"],
+    "prettier/prettier": [
+      "error",
+      {
+        "tabWidth": 2,
+        "semi": true,
+        "trailingComma": "es5",
+        "singleQuote": true
+      }
+    ],
+    "quotes": [2, "single"],
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "max-len": [2, { "code": 80 }]
+  },
+  "settings": {
+    "import/resolver": {
+      "alias": {
+        "map": [
+          ["Images", "./src/Images"],
+          ["Components", "./src/Components"]
```

Comment:
should be "./src/components" no?

Reply:
in the files the Components folder uses a capital C in the start. Should it have a lowercase c instead?

Reply:
In the version being reviewed here the folder has a lower case "c", therefore the alias was throwing an error.
Usually, folders always start with lowercase as well as most file names. Uppercase file names are reserved for react components. You should actually update that as well.

<img width="384" alt="Screenshot 2022-02-23 at 09 09 02" src="https://user-images.githubusercontent.com/75747772/155290154-588d4c4d-4122-4d93-95cc-5fe1f3ec286b.png">